libraries/shared/src/shared/Camera.h

```diff
@@ -181,6 +214,20 @@ public slots:
      */
     void modeUpdated(const QString& newMode);
 
+    /**jsdoc
+     * Triggered when the camera mouse capture state changes.
+     * @function Camera.captureMouseUpdated
+     * @param {boolean} newCaptureMouse - The new mouse capture state.
+     * @returns {Signal}
+     * @example <caption>Report mouse capture state changes.</caption>
+     * function onCaptureMouseUpdated(newCaptureMouse) {
+     *     print("The mouse capture has changed to " + newCaptureMouse);
+     * }
+     *
+     * Camera.captureMouseUpdated.connect(onCaptureMouseUpdated);
+     */
+    void captureMouseUpdated(bool newCaptureMouse);
```

Comment:
```suggestion
    void captureMouseChanged(bool newCaptureMouse);
```
`captureMouseChanged()` would be a better name for this signal. (Most such signals use "Changed" rather than "Updated".)